src/masternodeman.cpp

```diff
@@ -1609,22 +1606,17 @@ void CMasternodeMan::UpdateLastPaid(const CBlockIndex* pindex)
     IsFirstRun = false;
 }
 
-void CMasternodeMan::UpdateWatchdogVoteTime(const COutPoint& outpoint, uint64_t nVoteTime)
+void CMasternodeMan::UpdateLastSentinelPingTime()
 {
     LOCK(cs);
-    CMasternode* pmn = Find(outpoint);
-    if(!pmn) {
-        return;
-    }
-    pmn->UpdateWatchdogVoteTime(nVoteTime);
-    nLastWatchdogVoteTime = GetTime();
+    nLastSentinelPingTime = GetTime();
 }
 
-bool CMasternodeMan::IsWatchdogActive()
+bool CMasternodeMan::IsSentinelPingActive()
 {
     LOCK(cs);
     // Check if any masternodes have voted recently, otherwise return false
-    return (GetTime() - nLastWatchdogVoteTime) <= MASTERNODE_WATCHDOG_MAX_SECONDS;
+    return GetTime() - nLastSentinelPingTime <= MASTERNODE_SENTINEL_PING_MAX_SECONDS;
```

Comment:
I assume unary `-` takes precedence over `<=` operator? Not sure if explicit is better or cleaner code, but I do prefer cleaner (e.g. this change) TBH, as long as it's obvious. Just a comment/thought.

Note: this is in reference to removing the parens `()` around `(GetTime() - nLastWatchdogVoteTime)`

Reply:
Yep. It's usually a good idea to wrap the whole thing after `return` in `()` if it's too long or complex to avoid confusion though 🤔Will wrap it, let's see if it will be a bit more readable or not.